services/QuillLMS/app/workers/pre_cache_admin_dashboards_worker.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class PreCacheAdminDashboardsWorker
+  include Sidekiq::Worker
+  sidekiq_options queue: SidekiqQueue::DEFAULT
+
+  def perform
+    active_admins = User.where('last_sign_in >= ?', School.school_year_start(Time.now)).joins(:schools_admins).all
```

Comment:
Is `joins(:school_admins)` needed here?

Reply:
Also, what about `pluck(:id)` instead of all since we're just using the id?

Reply:
The `joins` is necessary because that's an inner join and means that we only pull users who have an entry in the `schools_admins` table, which is how we know that the user is an admin.  `pluck` is reasonable, so I'll make that change.